src/pspm_expand_epochs.m

```diff
@@ -0,0 +1,147 @@
+function [sts, ep_exp] = pspm_expand_epochs(epoches, expansion, options)
+
+% fn passt nicht zu missing epochs
+%   [sts, channel_index]   = pspm_expand_epochs( {data_fn,  channel},    expansion , options)
+%   [sts, output_file]     = pspm_expand_epochs( missing_epochs_fn,      expansion , options)
+%   [sts, expanded_epochs] = pspm_expand_epochs( missing_epochs,         expansion , options) %
+%   options.mode =  'datafile'
+%                   'missing_ep_file'
+%                   'missing_ep'
+
+global settings
+if isempty(settings)
+  pspm_init;
+end
+
+sts = -1;
+
+if nargin < 3
+  warning('ID:invalid_input', 'Not enough input arguments');
+  return;
+end
+
+switch options.mode
+    % missing epochs
+    case 'missing_ep'
+        % Directly expand the given epochs
+        [sts, ep_exp] = expand(epoches, expansion);
+        return;
+
+    % missing epoches file
+    case 'missing_ep_file'
+        % Load missing epochs from file
+        data = load(epoches);  % Load the file without specifying a variable
```

Comment:
Epochs should always be loaded through pspm_get_timing - this also checks the content of the file.